### PR TITLE
[Zeek] Add additional data sets

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Add new data sets for known_hosts, known_certs, known_services, & software logs files.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Relates #3288

### CODE REVIEW

**Comment by legoguy1000** [comment]
@efd6 can we retest

**Comment by legoguy1000** [comment]
fixed the test error. Ready to retest.

**Comment by legoguy1000** [comment]
@efd6 Any idea about the error from Elastic Machine or from Jenkins `found 0 hits in logs-zeek.known_certs-ep data stream: index_not_found_exception: no such index [logs-zeek.known_certs-ep] Status=404`??

**Review by efd6 — COMMENTED** [review]
Incidental to looking into the issue:

```
--- Test results for package: zeek - START ---
FAILURE DETAILS:
zeek/dns test-dns.log:
[0] parsing field value failed: the IP "40.126.31.143" is not one of the allowed test IPs (see: https://github.com/elastic/elastic-package/blob/main/internal/fields/_static/allowed_geo_ips.txt)
zeek/known_certs test-known_certs.log:
[0] parsing field value failed: field "event.kind"'s value "info" is not one of the allowed values (alert, enrichment, event, metric, state, pipeline_error, signal)
zeek/known_hosts test-known_hosts.log:
[0] parsing field value failed: field "event.kind"'s value "info" is not one of the allowed values (alert, enrichment, event, metric, state, pipeline_error, signal)
zeek/known_services test-known_services.log:
[0] parsing field value failed: field "event.kind"'s value "info" is not one of the allowed values (alert, enrichment, event, metric, state, pipeline_error, signal)
zeek/software test-software.log:
[0] parsing field value failed: field "event.kind"'s value "info" is not one of the allowed values (alert, enrichment, event, metric, state, pipeline_error, signal)
```

File: packages/zeek/data_stream/known_certs/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+# Sets event.created from the @timestamp field generated by filebeat before being overwritten further down
+  - set:
+      field: event.created
+      copy_from: "@timestamp"
+  - set:
+      field: ecs.version
+      value: '8.2.0'
+  - set:
+      field: event.kind
```
[the remark this answers is not in the document]
Comment (legoguy1000) [reply packages/zeek/data_stream/known_certs/elasticsearch/ingest_pipeline/default.yml:22]: Ya that should definitely be event.type, I'll fix it.

**Comment by efd6** [comment]
The non-progress issue comes down to the absence of terminating new-lines in the test inputs. There are bunch of other inputs that suffer the same issue, but these all have more than one test line, so they pass because only a single event is required to allow a system test to pass (they would have been found if [this](https://github.com/elastic/elastic-package/issues/821) were in place).

**Comment by legoguy1000** [comment]
🤦‍♂️ ya I've had this issue before, good catch. I'll fix this in the AM and then I think we should be good.

**Comment by efd6** [comment]
The other inputs that have the issue are 

- sample_logs/capture_loss.log
- sample_logs/dhcp.log
- sample_logs/files.log
- sample_logs/notice.log
- sample_logs/sip.log
- sample_logs/snmp.log
- sample_logs/ssh.log
- sample_logs/ssl.log

**Comment by legoguy1000** [comment]
@efd6 fixed

**Comment by legoguy1000** [comment]
@efd6 I think i got it now 🤦

**Comment by legoguy1000** [comment]
The `forwarded` tag needs to be added so that host is only set by zeek, not by the agent.

File: packages/zeek/data_stream/known_certs/_dev/test/system/test-logs-config.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+vars:
+  base_paths:
+    - "{{SERVICE_LOGS_DIR}}"
+input: logfile
+data_stream:
+  vars: ~
```
Comment (efd6) [inline packages/zeek/data_stream/known_certs/_dev/test/system/test-logs-config.yml:6]:
```suggestion
  vars:
    tags:
      - forwarded
```

File: packages/zeek/data_stream/known_hosts/_dev/test/system/test-logs-config.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+vars:
+  base_paths:
+    - "{{SERVICE_LOGS_DIR}}"
+input: logfile
+data_stream:
+  vars: ~
```
Comment (efd6) [inline packages/zeek/data_stream/known_hosts/_dev/test/system/test-logs-config.yml:6]: vars:
    tags:
      - forwarded

File: packages/zeek/data_stream/known_services/_dev/test/system/test-logs-config.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+vars:
+  base_paths:
+    - "{{SERVICE_LOGS_DIR}}"
+input: logfile
+data_stream:
+  vars: ~
```
Comment (efd6) [inline packages/zeek/data_stream/known_services/_dev/test/system/test-logs-config.yml:6]:
```suggestion
  vars:
    tags:
      - forwarded
```

File: packages/zeek/data_stream/software/_dev/test/system/test-logs-config.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+vars:
+  base_paths:
+    - "{{SERVICE_LOGS_DIR}}"
+input: logfile
+data_stream:
+  vars: ~
```
Comment (efd6) [inline packages/zeek/data_stream/software/_dev/test/system/test-logs-config.yml:6]:
```suggestion
  vars:
    tags:
      - forwarded
```

**Review by efd6 — APPROVED** [review]

**Pull request merged**